src/main/resources/i18n/aerospace/common.properties

```diff
@@ -175,6 +175,9 @@ LABELS_IN_CART=LABELS ABOUT CART
 LABELS_IN_LOCATIONS=LABELS ABOUT LOCATIONS
 LABELS_IN_ORDER_WIZARD=LABELS ABOUT ORDER WIZARD
 LABELS_IN_ORDER=LABELS ABOUT ORDER
+UPDATE_LABEL_REQUEST_ERROR = Bad request to update the labels.
```

Comment:
Please confirm with Roller about whether we need localization for Chinese. I see Renee did translation for some variables like "UPDATING_REQUEST_ERROR" in some earlier pull request.